### ISSUE TITLE
"switch segments" option not working

### ISSUE THREAD

**Issue by kchall** [PhonologicalCorpusTools/CorpusTools#635]
When segments are selected (e.g., for a ProD analysis), there's an option to switch the order of the segments in the function dialogue box. This works insofar as displaying the segments is concerned, but internally, the order hasn't been switched. 

E.g.:

IPHOD corpus; calculate ProD of [EY] and [EH] in the environment [-syll] __ #. 

The "natural" order of the segments is [EH], [EY], and using this order works -- there are 275 instances of [EY] and only 6 of [EH] (as expected for lax vowels in open syllables):

![image](https://cloud.githubusercontent.com/assets/7317450/23238677/ec7bd6e0-f917-11e6-9bfb-421e78bdd4bb.png)

![image](https://cloud.githubusercontent.com/assets/7317450/23238716/28f8fe54-f918-11e6-8146-1077f0e70c52.png)

But if we switch the order of the segments in the original dialogue box, we get the *same* results for the "second segment", which would mean we're getting 275 tokens of [EH], which is not accurate:

![image](https://cloud.githubusercontent.com/assets/7317450/23238736/4b20f180-f918-11e6-96d4-334ffaa46100.png)

![image](https://cloud.githubusercontent.com/assets/7317450/23238755/5dc1d502-f918-11e6-981c-4c0cbedfd93a.png)

**Comment by jsmackie** [comment]
After some testing, it seems like this bug is only cropping up when an environment is selected. Without environments (e.g. "FREQ-ONLY") the numbers looks correct.

**Comment by kchall** [comment]
Agreed that the underlying bug seems to be fixed. The order of columns in the output window is a bit odd, though: 

![image](https://cloud.githubusercontent.com/assets/7317450/24670633/aae53ef2-1923-11e7-8978-658a28055d0a.png)

In particular, the frequency of the second segment is listed first, and not adjacently to the frequency of the first segment.
